treeherder/push_health/push_health.py

```diff
@@ -102,16 +104,17 @@ def get_push_failures(push, option_map):
                 'platform': platform,
                 'config': config,
                 'key': test_key,
-                'jobs': [],
+                'failJobs': [],
+                'passJobs': [],
                 'logLines': [],
                 'suggestedClassification': 'New Failure',
                 'confidence': 0,
             }
             tests[test_key] = line
         test = tests[test_key]
         test['logLines'].append(failure_line.to_mozlog_format())
-        if not next((find_job for find_job in test['jobs'] if find_job['id'] == job.id), False):
-            test['jobs'].append(model_to_dict(job))
+        if not next((find_job for find_job in test['failJobs'] if find_job['id'] == job.id), False):
+            test['failJobs'].append(model_to_dict(job, fields=job_fields))
```

Comment:
what do we use the test object for?

Reply:
I'll add a comment in the code here, too.  But each ``test`` here is one of the test lines (test files) as displayed in Push Health view.